continuous_integration/install.sh

```diff
@@ -47,7 +47,15 @@ if [[ "$DISTRIB" == "conda" ]]; then
 
 elif [[ "$DISTRIB" == "ubuntu" ]]; then
     # Use standard ubuntu packages in their default version
-    sudo apt-get install -qq python-scipy python-nose python-pip
+    sudo apt-get install -qq python-scipy
+    # At the time of writing numpy 1.9.1 is included in the travis
+    # virtualenv but we want to used numpy installed through apt-get
+    # install.
+    deactivate
+    # Create a new virtualenv using system site packages for numpy and scipy
+    virtualenv --system-site-packages testvenv
+    source testvenv/bin/activate
+    pip install nose
```

Comment:
That's something I tried at one point and that failed for nilearn. The only reason I could fathom: the system nosetests ends up running as if you were outside the virtualenv. 

Note that using system nose should work for scikit-learn because you don't install any dependencies in the virtualenv (for nilearn nibabel is installed in the virtualenv). That does seem very appealing though.